sc/social/like/browser/viewlets.py

```diff
@@ -79,6 +80,7 @@ def update(self):
         self.site_name = portal.Title()
         self.language = facebook_language(get_language(self.context), 'en_US')
         self.image = get_content_image(self.context)
+        self.image_fallback = get_image_fallback(portal)
```

Comment:
we only need to get the fallback image if the content has no lead image; you have to load this later and only if needed.

Reply:
Ok